src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -30002,7 +30002,7 @@ public static int TakeOutParam(out int x, long y)
     }
 }
 ";
-            var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.ReleaseExe.WithScriptClassName("Script"), parseOptions: TestOptions.Script);
+            var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.ReleaseExe.WithScriptClassName("Script"), parseOptions: TestOptions.Script, skipUsesIsNullable: true);
```

Comment:
>skipUsesIsNullable [](start = 167, length = 18)

Is this a case where test depends on lazyily initializing symbols?
If not, should we have PROTOTYPE comment to remove `skipUsesIsNullable` later?

Reply:
Yes, these tests depend on lazily initializing symbols.

---
In reply to: [171085070](https://github.com/dotnet/roslyn/pull/25087#discussion_r171085070) [](ancestors = 171085070)